app/frontend/src/components/Markdown.tsx

```diff
@@ -56,7 +59,7 @@ export default function Markdown({
     return () => viewer.current?.remove();
   }, [source, topHeaderLevel]);
 
-  return <div className={classes.root} ref={rootEl} />;
+  return <div className={classNames(className, classes.root)} ref={rootEl} />;
```

Comment:
Extended this to allow some styling on this outer div container, mainly for adding spacing so things don't look terrible and hard to read with it stretching across the full width of the screen, so I don't need to add another wrapper div around this in the TOS